docs/image/sharp-logo.svg

```diff
@@ -1,5 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="86 86 550 550">
-<!-- Copyright 2019 Lovell Fuller. This work is licensed under the Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0) License. -->
-<path fill="none" stroke="#9c0" stroke-width="80" d="M258.411 285.777l200.176-26.8M244.113 466.413L451.44 438.66M451.441 438.66V238.484M451.441 88.363v171.572l178.725-23.917M270.323 255.602V477.22M272.71 634.17V462.591L93.984 486.515"/>
-<path fill="none" stroke="#090" stroke-width="80" d="M451.441 610.246V438.66l178.725-23.91M269.688 112.59v171.58L90.964 308.093"/>
-</svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="86 86 50 49.636" width="50" height="49.636" fill="none" stroke-width="7.275"><path d="M101.7 103.953l18.204-2.437M100.4 120.38l18.854-2.524m0 0V99.652m0-13.652v15.603l16.253-2.175m-32.724 1.78v20.154m.217 14.273v-15.603l-16.253 2.176" stroke="#9c0"/><path d="M119.264 133.46v-15.604l16.253-2.174m-32.782-27.478v15.603l-16.253 2.176" stroke="#090"/></svg>
```

Comment:
Please can the copyright and licensing notice remain in this file.

Reply:
Oh ! Missed it. Sorry about that.